src/config/dynamic-theme-fixes.config

```diff
@@ -1293,6 +1293,19 @@ span[data-testid="file-tree-file__comments"] > span {
 span.code {
     background-color: rgba(255, 247, 218, 0.08) !important;
 }
+[type="button"]:not(.css-1l4j2co, .css-1h2061w, [aria-expanded]) {
+    background-color: rgba(159, 179, 213, 0.24) !important;
+}
+[type="button"]:not(.css-1l4j2co, .css-1h2061w, [aria-expanded]):hover {
+    background-color: rgba(159, 179, 213, 0.48) !important;
+}
+.ak-navigation-resize-button {
```

Comment:
Where can I find the resize-button?

Reply:
If you are logged in, it is the arrow button that appears when you hover over the left panel:
![image](https://user-images.githubusercontent.com/5030061/112552994-fd22fd80-8dc3-11eb-8d22-d0680104b37e.png)